### DISCUSSION TITLE
IOS Version

### DISCUSSION THREAD

**Discussion by michelmelo** [gokadzev/Musify#304] [General]
Hello, I'm a programmer and I see that we don't have a version of the application in the Apple store. I am available to add the application through my account. what do you think?

**Comment by gokadzev** [comment]
@michelmelo Hello, yes, you're right. The iOS version of the app isn't available in the Apple Store or in general.

Will you be able to test it on either an emulator or a real device? Additionally, some changes are needed to make it work on iOS as well. If you have enough experience and can make the appropriate changes for the iOS version, please feel free to open a PR. Once it becomes usable, we can discuss adding it to the Apple Store or releasing IPA signed files for iOS users. (I'm not sure about Apple Store rules, particularly regarding the publication of YouTube-based apps).

Many people have asked for the release of an iOS version of the app before. However, due to time constraints, not having access to a Mac, and not having an account, I preferred not to extend support to different platforms and focused on primarily Android and partially Linux (Alpha). If you wish to take this task upon yourself and work on it (Ensure that all changes are visible in the repository and the app is built with the original source code, with no untracked changes), I will give you credits in the README as the iOS version developer and publish IPA files in the releases.

**Reply by michelmelo** [reply]
@gokadzev 
Good morning, I have to change the hive package because on iOS it is poorly configured. he the project cannot locate.

https://github.com/michelmelo/Musify

**Reply by gokadzev** [reply]
> @gokadzev Good morning, I have to change the hive package because on iOS it is poorly configured. he the project cannot locate.
> 
> https://github.com/michelmelo/Musify

@gokadzev Good morning, what alternative are you considering? (Also, in the pubspec, you added 'flutter_launcher_icons' in dependencies and its configuration, but it was already added in dev_dependencies and its configuration as well.)
